.github/workflows/docker-image-test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
               uses: docker/build-push-action@v2
               with:
                   push: false
-                  tags: posthog/posthog:latest
+                  tags: posthog/posthog:testing
```

Comment:
Is this a correct change? 100% unrelated to everything and quite suspect.

Reply:
Forgot about this, but yes it is safer this way, which is why I left it in. As Michael called out already for the cases when someone changes `push:true`, if they then push a broken package from their wip PR to `latest` it wouldn't be all that great as that can get picked up by our users.